php-nginx/etc/confd/templates/nginx/site.conf.tmpl

```diff
@@ -1,5 +1,5 @@
 server {
-    server_name _;
+    server_name {{ getenv "WEB_SERVER_NAME" }};
```

Comment:
what does this functionality give in a single nginx server context? or is it for if adding multiple?